rest_docs/rest-catalog-open-api-v0.1.yaml

```diff
@@ -0,0 +1,763 @@
+        required: true
+        schema:
+          type: string
+        example: "sales"
+    get:
+      tags:
+        - Catalog API
+      summary: Load a given table from a given namespace
+      operationId: loadTable
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/GetTableResponse'
+        '401':
+          description: Unauthorized
+        # Using 412, `Precondition Failed`, instead of 404, as 404 makes monitoring response codes from ELBs
+        #  very difficult - Hard to tell if clients or servers are misconfigured and calling non-existent routes
+        #  or missing routes versus expected error cases such as NoSuchTableException (expected meaning that
+        #  a person who is on call shouldn't be paged for this but 404 they might need to be).
+        '412':
+          description: NoSuchTableException
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/NoSuchTableError'
+    put:
+      tags:
+        - Catalog API
+      summary: Commit an in progress create (or replace) table transaction
+      operationId: commitTable
+      description: Commit a pending create (or replace) table transaction, e.g. for doCommit.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CommitTableRequest'
+        required: true
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/CommitTableResponse'
+        '401':
+          description: Unauthorized / Invalid Credentials
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/IcebergResponseObject'
+    delete:
+      tags:
+        - Catalog API
+      summary: Drop a table from the catalog, optionally purging the underlying data
+      operationId: dropTable
+      description: Remove a table from the catalog, optionally dropping the underlying data
+      parameters:
+        - name: purge
+          in: query
+          required: false
+          schema:
+            type: boolean
+            default: false
+      responses:
+        '200':
+          description: OK
+          content:
+            'application/json':
+              schema:
+                type: boolean
+    head:
+      tags:
+        - Catalog API
+      summary: Check if a table with a given identifier exists
+      operationId: tableExists
+      description:
+        Check if a table exists within a given namespace. Returns the standard response with `true` when found. Will return a TableNotFound error if not present. Can change to returning a 200 with a body of `false` if not found, but that does add more wok on the client.
```

Comment:
You can drop "within a given namespace" since that's implied by the required parameter.

Can you also remove the notes from the description? I don't think your intent was to leave "Can change to returning..." in the docs, right? We should have only one "right" way to implement this protocol.